src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -0,0 +1,236 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+"""
+    SemidiscretizationHyperbolicParabolic
+
+A struct containing everything needed to describe a spatial semidiscretization
+of a mixed hyperbolic-parabolic conservation law.
+"""
+
+struct SemidiscretizationHyperbolicParabolic{Mesh, Equations, ParabolicEquations, InitialCondition,
+                                             BoundaryConditions, ParabolicBoundaryConditions,
+                                             SourceTerms, Solver, Cache, ParabolicCache} <: AbstractSemidiscretization
+
+  mesh::Mesh
+
+  equations::Equations
+  parabolic_equations::ParabolicEquations
+
+  # This guy is a bit messy since we abuse it as some kind of "exact solution"
+  # although this doesn't really exist...
+  initial_condition::InitialCondition
+
+  boundary_conditions::BoundaryConditions
+  parabolic_boundary_conditions::ParabolicBoundaryConditions
+
+  source_terms::SourceTerms
+
+  solver::Solver
+  # TODO: do we want to introduce `parabolic_solver` for future specialization?
```

Comment:
We should at least have the option to choose different numerical parameters of the parabolic part. I need to read the rest of this PR before being able to comment better on this.

Reply:
For numerical parameters, are you referring to penalization terms? I'm not sure what other parameters show up.

Reply:
But yes, I agree we need a way to pass in parameters. I'll add that as a todo item.

Reply:
I was thinking for example about different numerical fluxes (Bassi-Rebay, LDG, ...)

Reply:
Yes; let me get LDG working first and then I can introduce a field for this (and maybe set LDG as default).